CHANGELOG.md

```diff
@@ -11,6 +11,8 @@
 
 ### Fixed
 
+- Fixed a bug that ignored the `--show-log-origin` option, preventing it from
```

Comment:
nit: can we add the PR number here?

Reply:
The original PR or the backport?

Reply:
it should be the backport PR so in this case 4277

Reply:
done